tests/foreman/cli/test_computeresource.py

```diff
@@ -159,6 +163,52 @@ def test_create_positive_libvirt(self, options):
         self.assertEqual(result.return_code, 0)
         self.assertEqual(len(result.stderr), 0)
 
+    def test_create_compute_resource_with_location(self):
+        """@Test: Create Compute Resource with location
+
+        @Feature: Compute Resource - Location Create
+
+        @Assert: Compute resource is created and has location assigned
+
+        """
+        try:
+            location = make_location()
+            comp_resource = make_compute_resource({
+                'location-ids': location['id'],
+            })
+        except CLIFactoryError as err:
+            self.fail(err)
+        self.assertEqual(1, len(comp_resource['locations']))
+        self.assertEqual(comp_resource['locations'][0], location['name'])
+
+    @skip_if_bug_open('bugzilla', 1214312)
+    @data(u'True', u'Yes', 1, u'False', u'No', 0)
+    def test_create_compute_resource_with_console_password(self, data):
```

Comment:
Please do not use a variable named `data`. You're redefining the `data` that is imported at the top of this module.